dashboard/src/components/common/Wizard.tsx

```diff
@@ -49,7 +47,7 @@ export function Wizard({
     <Modal
       icon={icon}
       title={title}
-      closeable={closeable ?? false}
+      closeable={(currentStep as any)?.props?.id !== 'execdbcreate'}
```

Comment:
I don't think this is the best solution. The `Wizard` component should not know about the specifics of the `Connect` component.

Reply:
I get your point and I'll refactor the code to not depend on the internals of its caller component. But the `Wizard` component has not been used anywhere else.